AutomatedTesting/Gem/PythonTests/assetpipeline/CMakeLists.txt

```diff
@@ -10,6 +10,19 @@ add_subdirectory(asset_processor_tests)
 add_subdirectory(fbx_tests)
 add_subdirectory(scene_settings_tests)
 
+    ly_add_pytest(
+        NAME AssetPipelineTests.Periodic
+        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Periodic.py
+        EXCLUDE_TEST_RUN_TARGET_FROM_IDE
```

Comment:
is EXCLUDE_TEST_RUN_TARGET_FROM_IDE a "cut and paste" thing, or is there a reason for it?

Reply:
It's a cut&paste thing; I just saw a previous conversation regarding it's use and will remove from this test as well.